NorthstarDLL/core/filesystem/filesystem.cpp

```diff
@@ -73,8 +73,6 @@ void SetNewModSearchPaths(Mod* mod)
 	{
 		if ((fs::absolute(mod->m_ModDirectory) / MOD_OVERRIDE_DIR).string().compare(sCurrentModPath))
 		{
-			NS::log::fs->info("Changing mod search path from {} to {}", sCurrentModPath, mod->m_ModDirectory.string());
```

Comment:
Unrelated to PR, can someone explain me the difference between `NS::log::fs` and `spdlog::info`?

I know what the later does, just kinda confused why we also had our custom logger... ^^"

Reply:
We have multiple different loggers for spdlog, all under `NS::log`.
`NS::log::fs` is for filesystem logs, and has the prefix `[FILESYSTM]` in logs
`NS::log::rpak` is for rpak filesystem logs, and has the prefix `[RPAK_FSYS]` in logs
etc. etc.

Basically its so we can get better context about where a line of logging came from. It was part of the log colours PR

Reply:
Aaah, makes sense. So, we use `spdlog` as generic log then? Or why the switch? ^^"

Reply:
Yeah basically. `spdlog` I believe just uses the default logger (which is setup to prefix with `[NORTHSTAR]`)
There may be an `NS::log` variant for the same job, but meh